package.json

```diff
@@ -25,21 +25,35 @@
   "main": "main.js",
   "bin": "bin/regenerator",
   "scripts": {
-    "pretest": "./link.sh",
+    "bootstrap": "lerna bootstrap",
     "test": "node test/run.js"
   },
   "dependencies": {
     "babel-core": "^6.18.2",
+    "babel-plugin-syntax-async-functions": "^6.13.0",
+    "babel-plugin-syntax-async-generators": "^6.13.0",
+    "babel-plugin-transform-es2015-arrow-functions": "^6.8.0",
```

Comment:
all of those new deps here got hoisted manually by me from each package, it is supposed to make process of bootstrapping + hoisting faster (lerna)